packages/core/src/domain/internalMonitoring/internalMonitoring.ts

```diff
@@ -48,12 +49,7 @@ export interface MonitoringMessage extends Context {
   }
 }
 
-const TELEMETRY_ALLOWED_SITES: string[] = [
-  INTAKE_SITE_US5,
-  INTAKE_SITE_US3,
-  INTAKE_SITE_EU,
-  // INTAKE_SITE_US,
-]
+const TELEMETRY_ALLOWED_SITES: string[] = [INTAKE_SITE_US5, INTAKE_SITE_US3, INTAKE_SITE_EU, INTAKE_SITE_US]
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Now that it's allowed on most DC, maybe we could have an exclude list to explicitly discard telemetry on fed for example?